feature-rich/index.js

```diff
@@ -22,6 +22,9 @@ const CKBOX_TOKEN_URL = '';
 
 // WProofreader plugin require a license key to work properly.
 // For more info how to get the key, see https://ckeditor.com/docs/ckeditor5/latest/features/spelling-and-grammar-checking.html.
+//
+// IMPORTANT: WProofreader has a mechanism preventing it to run on 'localhost' domain. To test it locally, you need to use
+// a different domain name, by, for example, mapping 'localhost' to 'fake.ckeditor.com' in your '/etc/hosts' file.
```

Comment:
```suggestion
// a different domain name, by, for example, mapping 'localhost' to 'localhost.dev' in your '/etc/hosts' file.
```

I would rather not leak fake.ckeditor.com.

Reply:
`.dev` domains can be locally? AFAIR when Google bought this makes some redirects even locally on Chrome? :thinking:

Maybe `dev.your-domain.com`?

Reply:
ah, you're right, I saw also `dev.localhost` pattern I think.

Reply:
Ok, I'll fix it 👍

I'm just confirming now how it works exactly. I see with trial key, all domains are allowed by default 🤔 Probably, that's not the case for paid plan. I'm checking this and will adjust the message if needed.

Reply:
Ok, I talked to @Stalq, and it seems this is no longer relevant. It worked like that some time ago but now pricing model is based on the number of spellchecked words (AFAIU).

There is a way to change allowed/excluded domains on WSC side to e.g. to exclude some, but not automatically (requires contacting their support) so this is a really custom thing.

I'll restore the original comments. Sorry for not checking this in more details earlier 😓 🙈